.storybook/stories/Float.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { forwardRef, Suspense, useRef } from 'react'
+import * as THREE from 'three'
+import { withKnobs, number } from '@storybook/addon-knobs'
+import { useFrame } from '@react-three/fiber'
+
+import { Setup } from '../Setup'
+
+import { Float } from '../../src'
+
+export default {
+  title: 'Staging/Float',
+  component: Float,
+  decorators: [withKnobs, (storyFn) => <Setup cameraPosition={new THREE.Vector3(0, 0, 10)}> {storyFn()}</Setup>],
+}
+
+function FloatScene() {
+  const cube = useRef()
+
+  return (
+    <>
+      <Suspense fallback={null}>
+        <Float
+          position={[0, 1.1, 0]}
+          minPosition={0}
+          maxPosition={number('Max Position', 4)}
```

Comment:
Maybe an array for `floatingRange={[ min, max ]}`? How does this work with intensity?

Reply:
the array sounds good `range={[min, max]}`.

I believe intensity acts just like it did, as a multiplier.

Reply:
Perhaps we can apply intensity after re-mapping? that would be more intuitive

Reply:
Okay let me do that today! :3

Reply:
I applied intensity in the re-mapping because the range basically means it won't float beyond those values. Like [2,-2] means the yPosition of that object would never go beyond 2 or -2.

Reply:
Nice! Did you push the changes? Don’t see them yet

Reply:
Also the reason I made it two separate props is because I wanted them to be optional. Like you can only pass the minPosition as a prop and leave the maxPosition as default. I don't know how to do that if it was an array like floatingRange={[ min, max ]}

Reply:
I applied floatIntensity after the mapping because ur right, its more intuitive as a multiplier

Reply:
With arrays you can do this:
```ts
THREE.MathUtils.mapLinear(
    yPosition, 
    -0.1, 
    0.1, 
    floatingRange?.[0] ?? -Infinity, 
    floatingRange?.[1] ?? Infinity
)
```

Reply:
oh :0 let me do that then

Reply:
so if they only want to set floatingRange[1] how do they pass that in as a prop ?

Reply:
They can do `floatingRange={[null, -1]}`

Reply:
Yeah thought so, I've done that in the storybook example! Let me know if there's anything else I need to add :D